src/app.js

```diff
@@ -15,15 +15,22 @@ const authRouter = require("./routes/authRouter");
 const { authenticateUser } = require("./middleware/authHandler");
 
 // api documentation: swagger-ui
-const swaggerDocument = require("yamljs").load("./src/swagger.yaml");
+const swaggerDocument = require("yamljs").load(
+  path.join(__dirname, "swagger.yaml")
+);
```

Comment:
If both you and @marice-romero change this line, you'll likely get a merge conflict. If you don't mind handling it, it's no problem to leave it, but it might be easier to just make the change in one PR.